di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/IdentityMapper.java

```diff
@@ -132,7 +131,8 @@ public SharedClaims mapToSharedClaim(Identity identity, boolean agedDOB) {
                 canonicalAddresses,
                 identity.nino() == null
                         ? null
-                        : List.of(new SocialSecurityRecord(identity.nino())));
+                        : List.of(new SocialSecurityRecord(identity.nino())),
+                null);
```

Comment:
Can we run the automation tests against your local IPV core stub which includes this code? Just want to be sure we're not breaking any existing mappings